### PR TITLE
fix(chart): Fix chart dataZoom bug and colors bug

### PR DESCRIPTION
**Pull request by Davont**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

## Summary by CodeRabbit

- **New Features**
	- Updated internal state management for chart options across multiple components to utilize `huiChartOption` instead of `option`.

- **Bug Fixes**
	- Improved consistency in handling chart configurations and options across various chart components.

- **Documentation**
	- Enhanced clarity in the codebase by renaming properties for better context and understanding.

- **Refactor**
	- Streamlined the naming convention for chart options to align with broader application standards, improving maintainability.

### CODE REVIEW

File: packages/vue/src/huicharts/huicharts-sunburst/src/chart-sunburst.vue [hunk 1]
```diff
@@ -28,7 +28,7 @@ export default {
       }
       const option = sunburst(columns, rows, this.settings, extra)
 
-      this.option = {
+      this.huiChartOption = {
         ...option
       }
```
[the remark this answers is not in the document]
Comment (Davont) [reply packages/vue/src/huicharts/huicharts-sunburst/src/chart-sunburst.vue]: 删除错误option定义，使用chart-core中的huiChartOption

[the remark this answers is not in the document]
Comment (Davont) [reply packages/vue/src/huicharts/huicharts-sunburst/src/chart-sunburst.vue]: 删除错误option定义，使用chart-core中的huiChartOption

[the remark this answers is not in the document]
Comment (Davont) [reply packages/vue/src/huicharts/huicharts-sunburst/src/chart-sunburst.vue]: 删除错误option定义，使用chart-core中的huiChartOption

File: packages/vue/src/huicharts/huicharts-line/src/chart-line.vue [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default {
       }
       const option = line(columns, rows, this.settings, extra)
 
-      this.option = {
+      this.huiChartOption = {
         smooth: true,
         ...option
       }
```
[the remark this answers is not in the document]
Comment (Davont) [reply packages/vue/src/huicharts/huicharts-line/src/chart-line.vue]: 删除多余smooth配置

File: packages/vue/src/huicharts/huicharts-histogram/src/chart-histogram.vue [hunk 3]
```diff
@@ -31,7 +31,7 @@ export default {
 
       const option = histogram(columns, rows, this.settings, extra, true)
 
-      this.option = {
+      this.huiChartOption = {
         smooth: true,
         ...option
       }
```
[the remark this answers is not in the document]
Comment (Davont) [reply packages/vue/src/huicharts/huicharts-histogram/src/chart-histogram.vue]: 删除错误option定义，使用chart-core中的huiChartOption

File: packages/vue/src/huicharts/huicharts-heatmap/src/chart-heatmap.vue [hunk 4]
```diff
@@ -40,7 +40,7 @@ export default {
         echartsLib: this.integrateChart.echartIns
       }
       const option = heatmap(columns, rows, this.settings, extra, true)
-      this.option = {
+      this.huiChartOption = {
         ...option
       }
```
[the remark this answers is not in the document]
Comment (Davont) [reply packages/vue/src/huicharts/huicharts-heatmap/src/chart-heatmap.vue]: 删除错误option定义，使用chart-core中的huiChartOption

File: packages/vue/src/huicharts/huicharts-radar/src/chart-radar.vue [hunk 5]
```diff
@@ -40,7 +40,7 @@ export default {
         extend: this.extend
       }
       const option = radar(columns, rows, this.settings, extra)
-      this.option = {
+      this.huiChartOption = {
         ...option
       }
```
[the remark this answers is not in the document]
Comment (Davont) [reply packages/vue/src/huicharts/huicharts-radar/src/chart-radar.vue]: 雷达图和进度图逻辑不同，进度图直接透传，而雷达图需要执行对应的处理函数

**Review by zzcr — APPROVED** [review]

**Pull request merged**